modules/dists/StencilDist.chpl

```diff
@@ -862,10 +849,26 @@ proc StencilDom.dsiBuildArray(type eltType, param initElts:bool) {
   return arr;
 }
 
-proc StencilDom.dsiNumIndices return whole.size;
-proc StencilDom.dsiLow return whole.low;
-proc StencilDom.dsiHigh return whole.high;
-proc StencilDom.dsiStride return whole.stride;
+// common redirects
+proc StencilDom.dsiLow           return whole.low;
+proc StencilDom.dsiHigh          return whole.high;
+proc StencilDom.dsiAlignedLow    return whole.alignedLow;
+proc StencilDom.dsiAlignedHigh   return whole.alignedHigh;
+proc StencilDom.dsiFirst         return whole.first;
+proc StencilDom.dsiLast          return whole.last;
+proc StencilDom.dsiStride        return whole.stride;
+proc StencilDom.dsiAlignment     return whole.alignment;
+proc StencilDom.dsiNumIndices    return whole.sizeAs(uint);
+proc StencilDom.dsiDim(d)        return whole.dim(d);
+proc StencilDom.dsiDim(param d)  return whole.dim(d);
+proc StencilDom.dsiDims()        return whole.dims();
+proc StencilDom.dsiGetIndices()  return whole.getIndices();
+//proc StencilDom.dsiMember(i)     return whole.contains(i);
```

Comment:
Is this intentionally commented out?

Reply:
Correct. In this and a couple of similar cases the domain map provides a different implementation for this feature that is elsewhere in the file.